Contributing-code-to-Oppia.md

```diff
@@ -18,6 +19,19 @@ If you run into any problems along the way, we're here to help! Check out our [[
 * [Tips for Success](#tips-for-success)
 * [Notes](#notes)
 
+## How to find information on this wiki
+
+1. In the top-right corner of the repository page, you'll find a search bar. Click on it to expand the search options.
+2. Enter the search query you want to use. You can search for specific keywords, file names, code snippets, or any other relevant information you're looking for.
+3. GitHub provides some advanced search filters that you can use to refine your search further. You can filter results by the file type, code language, author, or specific locations within the repository. To use these filters, click on the "Filters" button next to the search bar and select the desired options.
+4. Press Enter or click on the search button to initiate the search.
+5. GitHub will display the search results based on your query. The results will show matching files, code snippets, issues, pull requests, commits, discussions, wikis and other relevant information which you can see in left side after search.
+6. You can click on any search result to view more details or explore the code/files associated with it.
+
+Additionally, GitHub also provides an advanced search syntax that allows you to construct more complex queries. You can find more information about this syntax in the [GitHub documentation](https://docs.github.com/en/github-ae@latest/search-github/searching-on-github/searching-code).
+
+<img width="1074" alt="Screenshot wiki preview" src="https://github.com/MohitGupta121/oppia-android/assets/76530270/866c6796-3c50-41fe-992a-778728401c8f">
```

Comment:
Can you make this a video? Also, can you use the white mode for this? The gif and the dark mode seem to make the gif hard to see on some screens. In the gif make sure you go to the "Code" tab, "issues" tab, PR "tab" and the "wiki" tab

Reply:
Done.